runtime/src/accounts_db.rs

```diff
@@ -3162,7 +3172,7 @@ impl AccountsDB {
     fn flush_slot_cache(
         &self,
         slot: Slot,
-        mut should_flush_f: Option<&mut impl FnMut(&Pubkey, &Account) -> bool>,
+        mut should_flush_f: Option<&mut impl FnMut(&Slot, &Pubkey, &Account) -> bool>,
```

Comment:
nits: Because `Slot` is `Copy` and `u64`, I don't think we need to use `&` for it. But if you want consistency for all args to be `&`, I don't say this strongly. :)

Reply:
removed `Slot` entirely